sormas-backend/src/test/java/de/symeda/sormas/backend/sample/SampleFacadeEjbTest.java

```diff
@@ -464,4 +468,83 @@ public void testGetByCaseUuids() {
 		assertThat(samples, hasSize(3));
 		assertThat(samples, contains(sample, sample2, sample3));
 	}
+
+	@Test
+	public void testGetSimilarSamples() {
+		TestDataCreator.RDCF rdcf = creator.createRDCF();
+		UserDto officer = creator.createUser(rdcf, UserRole.SURVEILLANCE_OFFICER);
+		CaseDataDto caze = creator.createCase(officer.toReference(), creator.createPerson().toReference(), rdcf);
+
+		Date sampleDateTime1 = DateHelper.subtractDays(new Date(), 10);
```

Comment:
Please use fixed dates instead of current dates. Sometimes using current dates in tests leads to weird effects.